app/views/api/v2/smart_proxies/version.json.rabl

```diff
@@ -0,0 +1,10 @@
+object @smart_proxy
+
+extends "api/v2/smart_proxies/base"
```

Comment:
Not sure these fields should be added, they don't make a lot of sense at the same level in the response as the success/message fields.  (Sorry, I notice now this is also in the LDAP test rabl, it probably shouldn't be too.)